### ISSUE TITLE
Numbers with underscore in formatters.go causing build to fail (WSL Ubuntu)

### ISSUE THREAD

**Issue by mmmkkaaayy** [boyter/scc#220]
Hi, I'm not very proficient with Go, so feel free to close this if it sounds like a problem that only applies to my setup.

I tried to install `scc` using the instructions in the README. I'm using WSL1 with Ubuntu 18.04 on Windows 10.

`go get -u github.com/boyter/scc/`
```
# github.com/boyter/scc/processor
go/src/github.com/boyter/scc/processor/formatters.go:929:32: syntax error: unexpected _048_576 at end of statement
go/src/github.com/boyter/scc/processor/formatters.go:931:32: syntax error: unexpected _024_000 at end of statement
go/src/github.com/boyter/scc/processor/formatters.go:936:33: syntax error: unexpected _000_000 at end of statement
go/src/github.com/boyter/scc/processor/formatters.go:960:32: syntax error: unexpected _000_000 at end of statement
```

If I go into formatters.go and delete the underscores on the offending lines, the `go get` succeeds and I can use tool in `~/go/bin`

Here's some info about my setup:
`go version` (installed with `sudo apt install golang-go`)
```
go version go1.10.4 linux/amd64
```
`cat /etc/os-release`
```
NAME="Ubuntu"
VERSION="18.04.4 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.4 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

**Comment by boyter** [comment]
This is totally my fault. The minimum Go version needed now is actually 1.13, but the documentation said 1.10. Have fixed it in 7d515bab0c1c651d3a1a2ba892b217662de0dcc0 

Its due to number literals with underscores being added in 1.13, and I tend to use anything new if I can to ensure the compiled version has a good level of baseline performance you get on the new compiler.

Sorry about that. You will just need to get a version of Go >= 1.13 (I would suggest going with the latest though for the performance gains) and it should work.

**Comment by boyter** [comment]
Closing down as it should be considered all find now.
